### ISSUE TITLE
Should focus next/save button before back button

### ISSUE THREAD

**Issue by jelhan** [jelhan/croodle#292]
If navigating between the elements using tab, the *back* button gets focused before *next*/*save* button in poll creation form. It should be the other way around.

The underlying issue is the wrong order of HTML elements. It should be switched around and the visual order been altered using `.flex-row-reverse` ([flex direction](https://getbootstrap.com/docs/4.3/utilities/flex/#direction)).

This is an accessibility issue as it especially affects users that uses assistance technologies.
